lib/presentation/authentication/desktop/reset_password_view_desktop.dart

```diff
@@ -44,10 +44,7 @@ class _ResetPasswordViewDesktopState extends State<ResetPasswordViewDesktop> {
             const SizedBox(
               height: 24,
             ),
-            SvgPicture.asset(
-              'assets/desktop/logo_desktop.svg',
-              width: 220,
```

Comment:
Hi @kareemabdeen, in few places the width and height has been removed for a few assets, can you pls check them?